android/app/src/main/java/org/missingmaps/mapswipe/MainApplication.java

```diff
@@ -6,20 +6,10 @@
 import com.facebook.react.PackageList;
 
 import com.facebook.react.ReactApplication;
-import io.invertase.firebase.RNFirebasePackage;
+import com.facebook.react.ReactInstanceManager;
 import com.BV.LinearGradient.LinearGradientPackage;
 import com.reactnativecommunity.asyncstorage.AsyncStoragePackage;
-import io.sentry.RNSentryPackage;
-import com.reactnativecommunity.viewpager.RNCViewPagerPackage;
 import com.reactnativecommunity.netinfo.NetInfoPackage;
-import io.invertase.firebase.RNFirebasePackage;
```

Comment:
all these packages have been removed due to the upgrade of the firebase package? and are now covered by something else?

Reply:
Yes, see previous comments, except for notifications, everything else is still there, but these lines are not needed anymore.